frontend/src/components/Masthead/Search.tsx

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from "@material-ui/core/styles";
+import SearchIcon from "@material-ui/icons/Search";
+import { Input } from "../Input";
+import { theme as appTheme } from "../../constants";
+
+const useStyles = makeStyles((theme) => ({
+  search: {
+    position: "relative",
+    background: appTheme.colors.inputBackground,
+    marginRight: theme.spacing(2),
+    width: "100%",
+    [theme.breakpoints.up("sm")]: {
+      marginLeft: theme.spacing(3),
+      width: "auto",
+    },
+  },
+  searchIcon: {
+    padding: theme.spacing(0, 2),
+    height: "100%",
+    position: "absolute",
+    pointerEvents: "none",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  inputRoot: {
+    color: "inherit",
+  },
+  inputInput: {
+    padding: theme.spacing(1.3, 1, 1.3, 0),
+    paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
+    transition: theme.transitions.create("width"),
```

Comment:
Can't see that the transitions on width work

![transition_width](https://user-images.githubusercontent.com/33123587/113544441-8d451c00-95e8-11eb-8207-ed5f972ac29a.gif)

Reply:
@Madlenne  maybe I'm missing something. Do we have such a requirement that the app should be responsive? Desktops will not be such small, so are you suggesting we should support mobiles/tablets? If so please update the design to include all device sizes. Anyway I implemented really basic responsiveness, so I expect that all app views implemented in future will be responsive now.